docs/concepts/configuration/taint-and-toleration.md

```diff
@@ -171,15 +171,18 @@ hardware (e.g. `kubectl taint nodes nodename special=true:NoSchedule` or
 toleration to pods that use the special hardware. As in the dedicated nodes use case,
 it is probably easiest to apply the tolerations using a custom
 [admission controller](/docs/admin/admission-controllers/)).
-For example, the admission controller could use
-some characteristic(s) of the pod to determine that the pod should be allowed to use
-the special nodes and hence the admission controller should add the toleration.
-To ensure that the pods that need
-the special hardware *only* schedule onto the nodes that have the special hardware, you will need some
-additional mechanism, e.g. you could represent the special resource using
-[extended resources](/docs/concepts/configuration/manage-compute-resources-container/#extended-resources)
-and request it as a resource in the PodSpec, or you could label the nodes that have
-the special hardware and use node affinity on the pods that need the hardware.
+For example, you can use [Extended
```

Comment:
Mention that we *recommend* using the extended admission controller to guarantee portability for extended resources.

Reply:
@vishh I added a comment to this effect as a Note callout - Our style guide is set up to discourage the use of "we" (https://kubernetes.io/docs/home/contribute/style-guide/#avoid-using-we)

Reply:
Done. Added `it is recommended...`.